src/js/stores/NewsFeedStore.jsx

```diff
@@ -289,17 +287,16 @@ class NewsFeedStore extends EventEmitter {
     switch (action.type) {
       case 'NEWSFEED_GET': {
         this.dataFetched = true;
-        Array.prototype.push.apply(this.contentFeed.items, action.response.data.posts[0]);
+        this.contentFeed.items.push(...action.response.data.posts[0]);
         this.contentFeed.nextOffset = action.response.data.nextOffset;
         this.contentFeed.count = action.response.data.count;
         this.emit('updated');
         break;
       }
       case 'FLAGGED_GET': {
         this.dataFetched = true;
-        Array.prototype.push.apply(this.flaggedPosts.items, action.response.data.posts[0]);
-        this.contentFeed.nextOffset = action.response.data.nextOffset;
-        this.contentFeed.count = action.response.data.count;
```

Comment:
Yeah, Kyle doesn't give me a count (or an offset for that matter). Offset may show up in the future though